app/controllers/waivers_controller.rb

```diff
@@ -11,7 +11,9 @@ def new
     if @user.has_signed_waiver
       flash[:notice] = "You have already agreed to the release."
     end
-    
+
+    @should_see_video = !current_user.participant.is_scc? or params[:participant_id] == nil
```

Comment:
The user resolution logic already exists above in this method.  Should be `@should_see_video = @user.is_scc?`